src/server/app/util/templates/deactivate-email.server.view.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+
+<head></head>
+
+<body>
+<p>Hello  {{name}}, </p>
+
+<br>
+
+<p>It seems you haven't logged into your (WF | FH) account since {{date}}.</p>
+<p>We are emailing you to let you know that you have been inactive for 90 days. Therefore, your account has been deactivated.</p>
```

Comment:
Since this is a configurable threshold, can we calculate the number of days based on the last / highest configuration?